pkg/manifests/bundleloader.go

```diff
@@ -54,19 +54,32 @@ func (b *bundleLoader) calculateCompressedBundleSize() error {
 			if err != nil {
 				return err
 			}
+			if info.IsDir() {
+				return nil
+			}
+
+			if data, err := os.ReadFile(path); err == nil {
```

Comment:
I'm not sure if the non-nil `err` from `os.ReadFile` is being returned anywhere? Maybe on L84 we should return the error (nil if reading the file went well) or the non-nil error instead of simply nil

Reply:
Done